paddle/fluid/prim/utils/utils.cc

```diff
@@ -27,6 +27,14 @@ void PrimCommonUtils::SetBwdPrimEnabled(bool enable_prim) {
   StaticCompositeContext::Instance().SetBwdPrimEnabled(enable_prim);
 }
 
+bool PrimCommonUtils::IsEagerPrimEnabled() {
+  return StaticCompositeContext::Instance().IsEagerPrimEnabled();
+}
+
+void PrimCommonUtils::SetEagerPrimEnabled(bool enable_prim) {
+  StaticCompositeContext::Instance().SetEagerPrimEnabled(enable_prim);
+}
+
```

Comment:
it's better to add some describe for these flag

Reply:
Sure, i can do this in next pr